spec/requests/carto/api/visualizations_controller_spec.rb

```diff
@@ -1123,7 +1123,8 @@ def attributions_from_vizjson(visualization)
           {}, @headers
         last_response.status.should == 200
         overlays = JSON.parse(last_response.body)
-        overlays.length.should == 5
+        # One of the carto layers is hidden behind a feature flag in Bloomberg
```

Comment:
Which one? Actually if you could answer in the comment it would be great! Remember the housekeeping we'll need to undertake to maintain the fork, everything helps!

Reply:
The search overlay.  I have updated the comment in the code as well.